plugins/bulk-import/src/components/AddRepositories/RepositoryTableRow.tsx

```diff
@@ -72,23 +72,17 @@ export const RepositoryTableRow = ({
       </TableCell>
       <TableCell align="left" className={classes.tableCellStyle}>
         <Link to={data.repoUrl || ''}>
-          <>
-            {urlHelper(data?.repoUrl || '')}
-            <OpenInNewIcon
-              style={{ verticalAlign: 'sub', paddingTop: '7px' }}
-            />
-          </>
+          {urlHelper(data?.repoUrl || '')}
+          <OpenInNewIcon style={{ verticalAlign: 'sub', paddingTop: '7px' }} />
         </Link>
```

Comment:
When there is no repoUrl, what do we should here?

Instead of showing an empty link, we could maybe show just a dash?

```tsx
{ data.repoUrl ?
  (
    <Link .... />
  ) :
  "-"
}
```